.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
It's like `gitpod.yml`, but for GitHub Codespaces.
Your call. I think we could keep it since it's just one small file. Also, it's a standard so other cloud workspaces may eventually support it.